src/components/output/index.tsx

```diff
@@ -10,34 +14,88 @@ type State = {};
 
 export default class App extends Component<Props, State> {
   state: State = {};
-  canvas?: HTMLCanvasElement;
+  canvasLeft?: HTMLCanvasElement;
+  canvasRight?: HTMLCanvasElement;
+  pinchZoomLeft?: PinchZoom;
```

Comment:
Considering we render those in the JSX, can they ever be null?

Reply:
They're nullable in terms of TypeScript, since they're defined *after* the constructor.